main.py

```diff
@@ -269,6 +272,117 @@ def user_token(self, userid='core'):
+        response = requests.get(url_auth_check, headers={'Auth-Token': valid_token})
+        assert (response.status_code == 200) 
+        if "true" in response.text:
+            print("Check an authentication token's validity for VALID token - successfull connected")
+        
+        url_auth_check = urljoin(api_base_url, 'auth/check')
+        response = requests.get(url_auth_check, headers={'Auth-Token': invalid_token})
+        assert (response.status_code == 200)
+        if "false" in response.text:
+            print("Check an authentication token's validity for INVALID token - successfull connected")
+            
+        url_auth_update = urljoin(api_base_url, 'auth/update')
+        data = '''
+            {
+        "Auth-Token": "test",
+        "username": "test"
+            }
+        '''
+        response = requests.post(url_auth_update, data)
+        assert (response.status_code == 200)
+        print("Change the username - successfully connected")
+        
+        url_auth_public_key = urljoin(api_base_url, 'auth/public-key')
+        response = requests.get(url_auth_public_key)
+        assert (response.status_code == 200)
+        print("Receive authorization public key - successfully connected")
+        # Metastore service   
+        url_metastore_search = urljoin(api_base_url, 'metastore/search')
+        response = requests.get(url_metastore_search)
+        assert (response.status_code == 200)
+        print("Metastore search service - successfully connected")
+        
+    def check_frontend(self):
+        ''' Check fronend API'''
+        valid_token = self.config['JWT_TOKEN']
+        invalid_token = 'ivalid_token'
+        api_base_url = 'http://' + self.config['DOMAIN'] 
+        # Home page
+        url_home = urljoin(api_base_url, '/')
+        response = requests.get(url_home)
+        assert (response.status_code == 200)
+        print("Home page - successfull connected")
+        # Showcase page
+        url_showpage = urljoin(api_base_url, 'core/co2-ppm')
+        response = requests.get(url_showpage)
+        assert (response.status_code == 200)
+        print("Showcase page - successfull connected")
+        # Search page
+        # include query string
+        url_search = urljoin(api_base_url, 'search')
+        response = requests.get(url_search)
+        assert (response.status_code == 200)
+        print("Search page - successfull connected")
+        # Pricing page
+        url_pricing = urljoin(api_base_url, 'pricing')
+        response = requests.get(url_pricing)
+        assert (response.status_code == 200)
+        print("Pricing page - successfull connected")
+        # Owner page
+        url_owner = urljoin(api_base_url, 'core')
+        response = requests.get(url_owner)
+        assert (response.status_code == 200)
+        print("Owner page for valid publisher - successfull connected")
+        
+        url_owner = urljoin(api_base_url, 'dore')
```

Comment:
I would use something really nonexisting, some randomly generated. Some day this username may appear :)